src/components/CopiableWithTooltip.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import cn from 'classnames';
+import { ReactComponent as Copy } from 'assets/copy.svg';
+
+import useCopiable from 'lib/useCopiable';
+import WithTooltip from 'components/WithTooltip';
+
+export default function CopiableWithTooltip({
+  as: As = 'span',
+  text,
+  children,
+  className,
+  ...rest
+}) {
+  const [doCopy, didCopy] = useCopiable(text || children);
```

Comment:
When, if ever, does `didCopy` get cleared?

Reply:
it gets cleared after 2 seconds within the `useCopiable` logic